template-sample-app/template/src/data/sources/kpiSource.js

```diff
@@ -0,0 +1,19 @@
+export const KPI_SOURCE_ID = 'kpiSource';
```

Comment:
I'd simplify this to only export the default.

It adds a data structure that it's not required and it adds complexity in several parts of the code.

KPI_SOURCE_ID is available as default.id
KPI_SOURCE_COLUMNS is like have two ways to define the source at the SQL and the const. 

It's a key concept for new users and we need to simplify this as much as possible. 

If someone wants to complicate more to have a more robust architecture is fine, but not for all the users